opm/autodiff/FlowMainEbos.hpp

```diff
@@ -546,8 +543,7 @@ namespace Opm
             {
                 const EclipseGrid& inputGrid = eclState().getInputGrid();
                 eclipse_writer_.reset(new EclipseWriter(eclState(), UgGridHelpers::createEclipseGrid( grid , inputGrid )));
-                eclipse_writer_->writeInitial(geoprops_->simProps(grid),
-                                              geoprops_->nonCartesianConnections());
+                eclipse_writer_->writeInitial(computeLegacySimProps_(), eclState().getInputNNC());
```

Comment:
as far as I can see, this is not the case yet? (or at least I haven't stumbled over a deck which does this yet.)

also, this is just for output, right? simulator-wise "non-geometric" NNCs are not supported by flow_ebos anyway.

finally, shouldn't this functionallity be moved to the grid?